Microverse/style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [ ] As stated in the project requirement your .css file should includes one class with styles for your header (make it in your favorite color), Kindly update it.
![Capture](https://user-images.githubusercontent.com/93598189/197765215-b499f339-633e-4a34-a65f-8a5b9bbd5558.PNG)